packages/block-library/src/gallery/style.scss

```diff
@@ -147,4 +149,11 @@
 			justify-content: center;
 		}
 	}
+
+	+ figcaption {
```

Comment:
Where those styles should get applied? To the gallery's caption only? If yes, then it should be placed outside of `.blocks-gallery-grid`.

Reply:
Yup these are for the gallery caption only, and the problem here is that the RichText component won't allow me to add a classname to that element. Because there can potentially be other `figcaption`s inside the gallery, I'm targeting this one as `.blocks-gallery-grid + figcaption`. The plus sign between the two selectors makes it target only the figcaption that is the immediate next sibling to `.blocks-gallery-grid`.